src/client/routes/main-router.jsx

```diff
@@ -3,15 +3,16 @@ import React from "react";
 import { Route, withRouter, Link } from "react-router-dom";
 
 import Home from "./../Home";
+import StartVoyageMain from "../components/voyage/StartVoyageMain";
 
 const MainRouter = props => {
   return (
     <React.Fragment>
       <div className="container">
         <Route exact path="/" component={Home} />
+        <Route exact path="/StartVoyage" component={StartVoyageMain} />
```

Comment:
@zkwsk  I am not sure about the path /StartVoyage . Please confirm this.
Please check with Zaki about this.

Reply:
Comment resolved. Routing Path to StartVoyage UI has been removed from main-router file in React